cmd/kubectl/kubectl.go

```diff
@@ -17,17 +17,18 @@ limitations under the License.
 package main
 
 import (
-	"os"
-
 	"k8s.io/component-base/cli"
 	"k8s.io/kubectl/pkg/cmd"
+	"k8s.io/kubectl/pkg/cmd/util"
 
 	// Import to initialize client auth plugins.
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
 func main() {
 	command := cmd.NewDefaultKubectlCommand()
-	code := cli.Run(command)
-	os.Exit(code)
+	if err := cli.RunNoOutput(command); err != nil {
```

Comment:
RunNoErrOutput

Reply:
Darn. Well, now you know what my original name was before a last-minute change of mind...